easy-ui-react/src/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import {
+  Notification,
+  NotificationProps,
+  useNotification,
+} from "./Notification";
+import { Provider as EasyUIProvider } from "../Provider";
+import { Button } from "../Button";
+import { InlineStoryDecorator } from "../utilities/storybook";
+
+type Story = StoryObj;
+
+const meta: Meta<typeof Notification> = {
+  title: "Components/Notification",
+  argTypes: {},
+};
+
+export default meta;
+
+export const Notifications: Story = {
```

Comment:
as mentioned, each story gets its own `Provider` and hence their own queue. i found it best to group them in this way as to not pollute the console with warnings and to simulate the desired behavior that a new notification should end the cycle of the previous one. also, with regards to stacking and "priority",  based on my latest discussion, one notification at a time is what we want, but i will confirm/check again with design.